sem1/task1.py

```diff
@@ -0,0 +1,7 @@
+"""Sum of digits in three-digit number"""
+
+a = int(input('Enter a 3-digit number: '))
+if a < 100 or a > 999:
+    print('Please, enter correct number.')
+else:
+    print(f'The sum of digits is: {a % 10 + (a // 10) % 10 + a // 100}')
```

Comment:
У операций деления одинаковый приоритет
```suggestion
    print(f'The sum of digits is: {a % 10 + a // 10 % 10 + a // 100}')
```